apps/omg_watcher/lib/omg_watcher/block_getter.ex

```diff
@@ -46,6 +47,8 @@ defmodule OMG.Watcher.BlockGetter do
     GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
   end
 
+  def apply_block(%BlockApplication{} = to_apply), do: GenServer.cast(__MODULE__, {:apply_block, to_apply})
```

Comment:
why is this necessary? wouldn't it be better for the same execution to cast the message? dynamic function invocation is 6x slower then normal...

Reply:
at first, I thought that I cant decorate GenServer.cast/GenServer.call. 
And at this point, we could remove this.

Dynamic function? could You send me some article about why elixir execute this function so slow? 
I understand that this way will have one pater matching more. 
but that shouldn't make such a big difference

Reply:
http://erlang.org/doc/efficiency_guide/functions.html#function-calls